terraform/gitlab/ci-templates/k8s-cluster/.gitlab-ci.yml

```diff
@@ -206,3 +208,16 @@ destroy:
     - terragrunt run-all destroy --terragrunt-non-interactive -input=false
     - .gitlab/scripts/cleanapps.sh $CI_PROJECT_PATH $CI_SERVER_HOST $CI_COMMIT_REF_NAME $GITOPS_BUILD_OUTPUT_DIR $GITLAB_CI_PAT $ARGO_CD_ROOT_APP_PATH
   when: manual
+
+apps:
+  stage: deploy
+  resource_group: ${TF_STATE_NAME}
```

Comment:
@kalinkrustev what is this used for? the resource_group

Reply:
this was an error, removed